### PR TITLE
docs(tokens): NO-JIRA add border example for border-ai

### PR DESCRIPTION
**Pull request by ninamarina**
# docs(tokens): NO-JIRA add border example for border-ai

## :hammer_and_wrench: Type Of Change
The example for the token `var(--dt-color-border-ai)` was empty. This PR adds the example of the border with a gradient.

- [x] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :book: Jira Ticket
No Jira

## :camera: Screenshots / GIFs

| Before | Now |
|------|------|
| image | image |

### CODE REVIEW

**Review by juliodialpad — APPROVED** [review]
This is good from the engineering perspective, don't know if you need to wait for design approval. Thanks!

**Review by fede-dp — APPROVED** [review]
looks nice!

File: apps/dialtone-documentation/docs/.vuepress/baseComponents/tokens/TokenExample.vue
Comment (francisrupert) [inline apps/dialtone-documentation/docs/.vuepress/baseComponents/tokens/TokenExample.vue:119]: Add `border-color: transparent`. 

image

**Review by francisrupert — APPROVED** [review]
One little change, and approved.

Good workaround. CSS doesn't strictly support border gradients (yet), so this matches how an engineer would need to create the illusion of a gradient border.

**Pull request merged**
